### ISSUE TITLE
Crash with latest Vinery mod (Lets do Vinery) (1.20.1 Forge)

### ISSUE THREAD

**Issue by vivagabe** [TeamGalena/copperative#44]
### What happened?

https://www.curseforge.com/minecraft/mc-mods/vinery

Game crashes at start up. I informed the Lets Do Vinery dev as well.

[crash-2024-02-09_11.15.12-client.txt](https://github.com/Xaidee/coopperative/files/14227559/crash-2024-02-09_11.15.12-client.txt)
[latest(1).log](https://github.com/Xaidee/coopperative/files/14227560/latest.1.log)

### Mod Version

1.1.0

### Minecraft Version

1.20

**Comment by MagpieMaydin** [comment]
Yup! Me Too!
https://gist.github.com/MagpieMaydin/ec0b7df6c6e8cd3c9595d565eb6eb0b3

**Comment by burghenby** [comment]
Oh, that's handy that Github points out when an issue is mentioned in a different repository!
https://github.com/Team-Let-s-Do/DoAPI/issues/25

But yes, as it says above, I've also observed this conflict, though it _may_ be a bit more complex than just "Copperative and Vinery don't get along."

When I attempted to run just Copperative and Vinery in the hopes of getting a crash report just of them, my game loaded without issue. I subsequently went through a few rounds of adding back mods to figure out the true conflicts but ultimately just found that while other mods would result in crashes, when isolated with Copperative, the Let's Do mods, and themselves, the game would load fine. 

I think that there are multiple mods that aren't playing nice and it's just that certain mods, like Copperative in these cases, will tip the scale of compatibility and make it seem obvious that, "Hey, this mod isn't playing nice." For example, I found that when adding back mods to the both Copperative and Vinery, it was MoreMCMeta that caused a crash, but then when I ran it with just Copperative and Vinery the three ran without a crash. And then when I repeated the process I ran into the same thing with a different set of mods.

The best we can do is report what we find, dig as deep as our abilities allow, and then decide which mods are more important until things can be made compatible, if they can.

**Comment by vivagabe** [comment]
I havent tested too much but if you roll back to the previous version of copperative, it loads up and seems to run fine. the updated version doesnt seem to add or fix anything too important:

Russian Translation by @aveinzero
Add sound overwrite target to config
make spotlight replaceable
Add blocks/items in sorted order to creative tab

**Comment by NumanSadik** [comment]
I'm having this problem too, I removed the ones that are not affecting the crash to happen and I am left with these mods. Removing any of them solves the crash for some reason.
https://cdn.discordapp.com/attachments/231527759279685634/1259202708418854952/modpack.zip?ex=669759a8&is=66960828&hm=f7e22afbf45321c6da5331adf13c64783eb0f7f67f88e386eafc7b35c5753edb&

**Comment by vivagabe** [comment]
Have you tried the previous release of copperative (1.0.1)? it seems to work fine with all my mods (430 mods)

**Comment by NumanSadik** [comment]
Yes it does indeed work with the version 1.0.1 but I'd like to be able to update my mod and don't miss out on new features.
